apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -287,24 +365,25 @@ type FailedResourcePlacement struct {
 	Condition metav1.Condition `json:"condition"`
 }
 
-// ResourcePlacementConditionType defines a specific condition of a resource placement.
+// ResourcePlacementConditionType defines a specific condition of the cluster resource placement.
```

Comment:
Do we use this for the per CRP condition, the per cluster condition AND the per resource+cluster condition?